interp/notation_ops.ml

```diff
@@ -273,6 +273,10 @@ type found_variables = {
 let add_id r id = r := { !r with vars = id :: (!r).vars }
 let add_name r = function Anonymous -> () | Name id -> add_id r id
 
+let mkNApp1 (g,a) = match g with
```

Comment:
New function: please document it.

Reply:
It is somehow a self-explanatory function with only limited use. I'm not opposed to document it but I'm afraid that if we start to document this function, we would have by consistency to document all similar short functions and this would be much more work than this PR.

We could change the name to make it more explanatory (adding "flatten"??), but then, the inconsistency would be with all similar `mkApp` functions in the code.